src/SDKs/Azure.Core/data-plane/Azure.Core/Pipeline/HttpHeader.cs

```diff
@@ -49,10 +49,9 @@ public bool Equals(HttpHeader other)
 
         public static class Names
         {
-            public static string Host => "Host";
-            public static string TransferEncoding => "Transfer-Encoding";
-            public static string ContentLength => "Content-Length";
+            public static string Date => "Date";
```

Comment:
>public static string Date => "Date"; [](start = 12, length = 36)

I believe some services also use the x-ms-date header, should we support both interchangeably?

Reply:
Is `x-ms-date` being used for responses?

Reply:
Storage takes either for requests, but only returns Date as a response.